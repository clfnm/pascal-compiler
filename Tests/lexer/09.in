20 div 5
21 mod 5
20/5
20*5
2..