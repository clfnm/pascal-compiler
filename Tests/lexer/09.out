1	1	Integer	20	20
1	4	Keyword	DIV	div
1	8	Integer	5	5
2	1	Integer	21	21
2	4	Keyword	MOD	mod
2	8	Integer	5	5
3	1	Integer	20	20
3	3	Operation	DivisionWithAssigment	/
3	4	Integer	5	5
4	1	Integer	20	20
4	3	Operation	MultiplicationWithAssigment	*
4	4	Integer	5	5
5	1	Double	2	2.
5	3	Separator	Dot	.
5	4	Eof	EOF	